assessment/cog-memory-leaks.robot

```diff
@@ -0,0 +1,104 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# Required setup keywords:
+# Prepare Test Suite - elementary setup keyword for all tests.
+# Upload Required Images - uploads all required files onto the PiKVM.
+# Required teardown keywords:
+# Log Out And Close Connection - elementary teardown keyword for all tests.
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keywords
+...                     Log Out And Close Connection
+# Before the tests are executed the platform must be booted into Linux and have
+# Weston running in the background. The user should be logged in as root.
```

Comment:
@PLangowski, does the required state is achieved using one of the keywords? If not, this can be implemented as a "user input pause", introducing chosen keyword from Dialog library: https://robotframework.org/robotframework/latest/libraries/Dialogs.html#library-documentation-top

Reply:
242cff74a06b9f6db2c258471f98701ce89af088

Reply:
Note that some Python binaries don't have support for `tkinter`, which is used here.

Reply:
@PLangowski, why should we rely on manual steps if we can easily automate all steps/commands used within `Pause Execution` keywords? Some are even ready to use, like booting and logging to Linux. Use our keywords database, where you can use search-bar for your convenience: https://dasharo.github.io/open-source-firmware-validation/

Reply:
Booting to Linux requires Dasharo fw and logging in requires a password for the user, which is not the case on my DUT. I was, however, able to automate the other steps. e4bbd9c2869d2e2fd729e5da503e7f5f349a5565

Reply:
@PLangowski I have added an issue to get back to that in the future: https://github.com/Dasharo/open-source-firmware-validation/issues/516

Could you run the test and confirm whether the code with recent changes runs without errors?

Reply:
@PLangowski about tkinter and README: https://github.com/Dasharo/open-source-firmware-validation/blame/develop/README.md#L136

Reply:
@artur-rs I have tested it and it works